### PR TITLE
enhance(apps/website): add tags for competencies on about page

### PR DESCRIPTION
**Pull request by jajakob**

### CODE REVIEW

File: apps/website/src/components/common/Contact.tsx [hunk 1]
```diff
@@ -13,18 +13,25 @@ function Contact({ name, institution, role, link, imgSrc }: any) {
...
+            target="_blank"
+            href={link}
+            rel="noreferrer"
+          >
+            Profile
+          </a>
+        </div>
+        <div className="flex flex-col justify-end">
+          {tags.map((tag) => (
+            <div className="text-xs text-slate-700">{tag}</div>
```
Comment (rschlaefli) [inline apps/website/src/components/common/Contact.tsx]: I would style them as actual "tags" (e.g., like on the game cards) :) Right now they look a bit lost and they do not use the space efficiently (go down, not in a row)

File: apps/website/src/pages/about.tsx [hunk 2]
```diff
@@ -149,6 +150,7 @@ function About() {
             role=""
             link="https://www.le.uzh.ch/en/about-us/Community/davidschmocker"
             imgSrc={DavidImage}
+            tags={['Didactics']}
```
Comment (rschlaefli) [inline apps/website/src/pages/about.tsx]:
```suggestion
            tags={['Didactics', 'Psychology', 'Ethics']}
```

File: apps/website/src/pages/about.tsx [hunk 3]
```diff
@@ -141,6 +141,7 @@ function About() {
             role="Managing Director and Head of Teaching"
             link="https://www.df.uzh.ch/en/people/staff/management/benjamin-wilding.html"
             imgSrc={BenjaminImage}
+            tags={['Finance', 'Courses', 'Didactics']}
```
Comment (rschlaefli) [inline apps/website/src/pages/about.tsx]:
```suggestion
            tags={['Finance', 'Modelling', 'Didactics']}
```

File: apps/website/src/pages/about.tsx [hunk 4]
```diff
@@ -157,6 +159,7 @@ function About() {
             role="Program and Project Manager"
             link="https://www.df.uzh.ch/en/people/staff/teaching-center/anja-zgraggen.html"
             imgSrc={AnjaImage}
+            tags={['Finance', 'Courses']}
```
Comment (rschlaefli) [inline apps/website/src/pages/about.tsx]:
```suggestion
            tags={['Finance', 'Didactics', 'Curriculum Design']}
```

File: apps/website/src/pages/about.tsx [hunk 5]
```diff
@@ -165,6 +168,7 @@ function About() {
             role="IT Project Manager and Developer"
             link="https://www.df.uzh.ch/en/people/staff/teaching-center/roland-schlaefli.html"
             imgSrc={RolandImage}
+            tags={['IT', 'Artificial Intelligence', 'Games']}
```
Comment (rschlaefli) [inline apps/website/src/pages/about.tsx]:
```suggestion
            tags={['Software Engineering', 'AI', 'Modelling']}
```

File: apps/website/src/pages/about.tsx [hunk 6]
```diff
@@ -173,6 +177,7 @@ function About() {
             role="Computer Graphics and Game Software Engineer"
             link="https://www.df.uzh.ch/en/people/staff/teaching-center/jakob-jakob.html"
             imgSrc={JakobImage}
+            tags={['Computer Graphics', 'Games', 'Animations']}
```
Comment (rschlaefli) [inline apps/website/src/pages/about.tsx]:
```suggestion
            tags={['Software Engineering', 'Computer Graphics', 'Animations']}
```

**Review by rschlaefli — COMMENTED** [review]
I made suggestions for the tags (to make them a bit more consistent but showcase some individual "strengths" also)

**Comment by jajakob** [comment]
Screenshot 2024-06-19 at 14 02 28
Screenshot 2024-06-19 at 14 02 40
Screenshot 2024-06-19 at 14 02 58

**Pull request merged**